index.js

```diff
@@ -121,6 +123,41 @@ class Executor {
     stats() {
         return {};
     }
+
+    /**
+     * Get build JWT with temporal JWT using API and replace config.token
+     * @method exchangeTokenForBuild
+     * @param  {Object}  config          A configuration object
+     * @param  {String}  config.apiUrl   Base URL for Screwdriver API
+     * @param  {String}  config.buildId  Build ID
+     * @param  {String}  config.token    Temporal JWT
+     * @param  {String}  buildTimeout    Build timeout value which will be JWT expires time
+     */
+    async exchangeTokenForBuild(config, buildTimeout = DEFAULT_BUILD_TIMEOUT) {
+        try {
+            if (isFinite(buildTimeout) === false) {
+                throw new Error(`Invalid buildTimeout value: ${buildTimeout}`);
+            }
+
+            const options = {
+                uri: `${config.apiUri}/v4/builds/${config.buildId}/token`,
+                method: 'POST',
+                body: { buildTimeout },
+                headers: { Authorization: `Bearer ${config.token}` },
+                strictSSL: false,
+                json: true
+            };
+
+            const response = await request(options);
+
+            if (response.statusCode !== 200) {
+                throw new Error(`Failed to exchange build token: ${JSON.stringify(response.body)}`);
+            }
+            config.token = response.body.token;
+        } catch (err) {
+            throw new Error(err);
```

Comment:
I don't know we need this catch or not...

Reply:
I deleted try/catch.